### ISSUE TITLE
Rest Calls failing for gp-atom using gp-js-client

### ISSUE THREAD

**Issue by hchawla1** [IBM-Bluemix/gp-js-client#67]
Upload Bundle, Download Bundle, Create Bundle and Delete bundle failing in gp-atom when using gp-js-client. All functions (from gp-js-client) are failing when used in gp-atom. 

The root cause is REST calls are made even before credentials are verified using gp-hmac.

**Comment by hchawla1** [comment]
Scheme not set for Swagger https://github.ibm.com/gp/gp-rest/issues/243

**Comment by srl295** [comment]
> Scheme not set for Swagger

could not reproduce with node in console:
```
> const client = require('./index.js').getClient(require('./local-credentials.json')); client.ping(() => console.log(client.swaggerClient.scheme));
…
https
```

**Comment by hchawla1** [comment]
@srl295 @yumaoka 
I added the similar code in ATOM (gp-atom) was able to reproduce the issue. So, it is problem with ATOM. 

I added it to Download Bundle
https://github.com/IBM-Bluemix/gp-atom-package/blob/master/lib/gp-download-bundle-view.coffee

client = gaas.getClient(@credentials);
client.ping(() => console.log(client.swaggerClient.scheme));

On console.log I see "file" If I point to PROD environment and I see "https" when I point to DEV environment.
